redis/asyncio/lock.py

```diff
@@ -162,7 +172,13 @@ async def __aenter__(self):
         raise LockError("Unable to acquire lock within the time specified")
 
     async def __aexit__(self, exc_type, exc_value, traceback):
-        await self.release()
+        try:
+            await self.release()
+        except LockNotOwnedError as e:
+            if self.raise_on_release_error:
+                raise e
```

Comment:
```suggestion
                raise
```
to retain the original traceback?

Reply:
Oh interesting. Thanks!
`raise e` would not?